updater/includes/class-admin.php

```diff
@@ -0,0 +1,437 @@
+	}
+
+	/**
+	 * Method to load ThickBox and register the update notification hook.
+	 *
+	 * @access public
+	 *
+	 * @return void
+	 */
+	public function loadThemesScreen()
+	{
+		add_thickbox();
+		add_action('admin_notices', [$this, 'renderUpdateNag']);
+	}
+
+	/**
+	 * Method to register license page under the appearance menu.
+	 *
+	 * @access public
+	 *
+	 * @return void
+	 */
+	public function registerLicensePage()
+	{
+		$this->licensePageHook = add_submenu_page(
+			'capsule',
+			__('License', 'capsule'),
+			__('License', 'capsule'),
+			'manage_options',
+			'capsule-license',
+			[$this, 'renderLicensePage']
+		);
+	}
+
+	/**
+	 * Method to render license page.
+	 *
+	 * @access public
+	 *
+	 * @return void
+	 */
+	public function renderLicensePage()
+	{
+		$options = Options::get();
+		?>
+		<div class="wrap">
+			<h2><?php esc_html_e('Capsule License', 'capsule'); ?></h2>
+			<form action="" method="POST">
+				<?php wp_nonce_field('save_capsule_license_options', '_capsule_sl'); ?>
+				<table class="form-table">
+					<tbody>
+						<tr>
+							<th scope="row">
+								<label for="edd-license">
+									<?php esc_html_e('Enter Your License', 'capsule'); ?>
+								</label>
+							</th>
+							<td>
+								<input
+									id="edd-license"
+									class="regular-text"
+									type="password"
+									value="<?php echo esc_attr($options['license']); ?>" name="options[license]"
+								>
+								<br />
+								<div class="capsule-sl-field capsule-sl-field--checkbox">
+									<label for="capsule-sl-field-license-reveal">
+										<input type="checkbox" id="capsule-sl-field-license-reveal" value="0" />
+										<?php esc_html_e('Reveal license', 'capsule'); ?>
+									</label>
+								</div>
+								<?php include WP_CAPSULE_UPDATER_DIR . '/views/license-buttons.php'; ?>
+								<div class="capsule-sl-field capsule-sl-status capsule-sl-success license-status"
+									style="display: none;"></div>
+							</td>
+						</tr>
+					</tbody>
+				</table>
+			</form>
+		<?php
+	}
+
+	/**
+	 * Method to render the update notifications.
+	 *
+	 * @access public
+	 *
+	 * @return void
+	 */
+	public function renderUpdateNag()
+	{
+		$api_response = Client::getInstance()->getThemeUpdateTransient();
+
+		if (false === $api_response || version_compare(WP_CAPSULE_THEME_VERSION, $api_response->new_version, '>=')) {
+			return;
+		}
+
+		$theme = wp_get_theme(WP_CAPSULE_THEME_SLUG);
+
+		$update_url = wp_nonce_url(
+			'update.php?action=upgrade-theme&amp;theme=' . urlencode(WP_CAPSULE_THEME_SLUG),
+			'upgrade-theme_' . WP_CAPSULE_THEME_SLUG
+		);
+
+		$update_notice = __(
+			"Updating this theme will lose any customizations you have made. 'Cancel' to stop, 'OK' to update.",
+			'capsule'
+		);
+
+		$update_onclick = ' onclick="if(confirm(\'' . esc_js($update_notice) . '\')){return true;}return false;"';
+
+		// phpcs:disable Generic.Files.LineLength.TooLong
+		$update_available = __(
```

Comment:
`esc_html__` instead of `__`. Considering the need for html tags maybe wrap it around a  `wp_kses_post`?